backend/de.metas.business.rest-api-impl/src/test/resources/de/metas/rest_api/v1/ordercandidates/impl/OrderCandidatesRestControllerImpl_createOrderLineCandidates_Test.snap.debug

```diff
@@ -0,0 +1,3300 @@
+de.metas.rest_api.v1.ordercandidates.impl.OrderCandidatesRestControllerImpl_createOrderLineCandidates_Test.CreateProductPrice_WarehouseDestId=[
```

Comment:
pls remove this file

Reply:
also pls add an `**/*.snap.debug` rule to .gitignore

Reply:
ups